src/playsim/d_player.h

```diff
@@ -88,6 +88,11 @@ void PlayIdle(AActor *player);
 enum
 {
 	PPF_NOTHRUSTWHENINVUL = 1,	// Attacks do not thrust the player if they are invulnerable.
+	PFF_VIEWABSOFFSET =		1 << 3,
+	PFF_VIEWABSPOS =		1 << 4,
+	PFF_VIEWABSANGLE =		1 << 5,
+	PFF_VIEWABSPITCH =		1 << 6,
+	PFF_VIEWABSROLL =		1 << 7,
```

Comment:
There is an inconsistency here. The supplied flag is PPF_*, the new addition here is PFF_*

Reply:
Thanks for pointing that out! I completely overlooked that. And it helped me fix another small bug with the roll.